docs/HowToFAQ/Errors-Parser.md

```diff
@@ -1022,13 +1023,11 @@ are grouped. The above example should be written as either `(5 | 6) & 7` or `5 |
 
 ## **Error: too many characters in character literal**
 
+<!-- %check-resolve --unicode-char:false -->
 ```dafny
-const c := '\U{103456}';
+const c := '🚀';
```

Comment:
How does this illustrate the error type _too many characters in character literal_? What is your issue with the current example? Please check that the links to more detail below this part are correct.

Reply:
This kind of character is one that is not supported by in --unicode-char:false. I've asked Robin for text to put in as explanation.

Reply:
Thanks for clarifying